lib/exception_hunter/notifications_sender.rb

```diff
@@ -0,0 +1,9 @@
+module ExceptionHunter
+  module NotificationsSender
+    def self.notify(error)
+      ExceptionHunter::Config.notifiers.each do |notifier|
+        ExceptionHunter::SendSlackNotificationJob.perform_later(error, notifier.to_json) if notifier[:name] == :slack
```

Comment:
Why do you need to do `to_json`?

Reply:
I did this because it says I cannot pass symbols as parameters to a job. But maybe another way is to just stringify the keys? wdyt?